src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -49,9 +49,6 @@ final class ClassAttributesSeparationFixer extends AbstractFixer implements Conf
      */
     public const SPACING_ONE = 'one';
 
-    private const SUPPORTED_SPACINGS = [self::SPACING_NONE, self::SPACING_ONE];
-    private const SUPPORTED_TYPES = ['const', 'method', 'property', 'trait_import'];
```

Comment:
why dropping the constants?
(like, plenty of fixers are having possible options as constants, plenty of fixers are not. maybe we should align which approach we want to take :D )

Reply:
i see slight value in having it as `@internal public const` for re-usability in tests

Reply:
I think we should avoid using these constants in tests: if the value of a constant is changed it is a BC break but the tests won't notice. Then if the constant is used only once, it can be inlined unless the meaning of its value is not obvious, in which case the constant helps make it explicit.